docs/manifest/resources.md

```diff
@@ -49,6 +49,14 @@ The following additional sizes are also supported, but not required.
 - 48x48
 - 64x64
 
+The following image/file extensions are supported:
+- bmp
+- gif
+- exif
+- jpg
+- tiff
+- png
```

Comment:
Changing to confirm to style standards. Also, I alphabetized the list, but were those formats in that order for a reason?

```suggestion
The following image file formats are supported.
- BMP
- EXIF
- GIF
- JPG
- PNG
- TIFF
```